.vscode/launch.json

```diff
@@ -13,15 +13,16 @@
 			],
 			"preLaunchTask": "build-and-watch-vscode",
 			"env": {
-				"LogLevel": "warn"
+				"LogLevel": "warn",
+				"InstallDevPlugins": "false"
 			}
 		},
 		{
 			"request": "launch",
 			"name": "Launch CLI",
 			"type": "pwa-node",
 			"program": "${workspaceFolder}/index.ts",
-			"cwd": "/Users/jimmychenchen/Desktop/bugbug",
+			"cwd": "${workspaceFolder}",
```

Comment:
I don't think it points to the right folder, because now it points to the Taqueria repo. I think we should leave it `/Users/jimmychenchen/Desktop/bugbug` and maybe and a comment that say: "modify this to point to your taqueria project" or something.

Reply:
`cwd` is supposed to point to the Taqueria project that you're calling `taq` from. That's the right place to point to, which depends on the developer debugging. It's not ideal for now because developers always have to change it to their own folder, but idk how to parametrize this.

Reply:
But for me, I'm always calling from the same project that I work on.
If we keep this as `/Users/jimmychenchen/Desktop/bugbug`, it will fail for most users.

Reply:
Hmm that's ok but maybe make a comment that says "modify to point to where you're invoking taq. e.g. in a Taqueria project"

Reply:
OK. Give me a moment

Reply:
Reason I tend to wanna leave it like this is that it's more indicative of "point to where your taq project is"